spec/forms/idv/api_document_verification_status_form_spec.rb

```diff
@@ -8,12 +8,14 @@
     )
   end
 
-  let(:async_state) { ProofingDocumentCaptureSessionResult.none }
+  let(:async_state) { DocumentCaptureSessionAsyncResult.new }
   let(:document_capture_session) { DocumentCaptureSession.create! }
 
   describe '#valid?' do
     context 'with timeout async state' do
-      let(:async_state) { ProofingDocumentCaptureSessionResult.timed_out }
+      let(:async_state) {
+        DocumentCaptureSessionAsyncResult.new(status: DocumentCaptureSessionAsyncResult::TIMED_OUT)
+      }
```

Comment:
```suggestion
      let(:async_state) do
        DocumentCaptureSessionAsyncResult.new(status: DocumentCaptureSessionAsyncResult::TIMED_OUT)
      end
```

style nit... `do....end` for multiline blocks?